fluent-plugin-google-cloud.gemspec

```diff
@@ -14,7 +14,7 @@ eos
   gem.authors       = ['Todd Derr', 'Alex Robinson']
   gem.email         = ['salty@google.com']
 
-  gem.required_ruby_version = '~> 2.0'
```

Comment:
oops - I didn't notice you had that in there.  I looked at a few other gems which used the Requirement::new syntax (despite using the shorter syntax later in the gemspec), but I wasn't able to determine the justification for doing it.

Either way is fine with me but I'd rather use >= 2.0 than ~>